geth/node/manager.go

```diff
@@ -36,13 +43,14 @@ var (
 // should be fixed at https://github.com/status-im/status-go/issues/200
 type NodeManager struct {
 	sync.RWMutex
```

Comment:
Even if it's older, does the `RWMutix` API has to be public? Would prefer a private `mu sync.RWMutex` to avoid access from outside.

Reply:
This is abit out of scope for this PR, could you add your comment to https://github.com/status-im/status-go/issues/403 as well, so the handler for that issue, knows to make this change.